web-app/package.json

```diff
@@ -0,0 +1,62 @@
+{
+	"name": "web-app",
+	"version": "0.1.0",
+	"private": true,
+	"dependencies": {
+		"@emotion/react": "^11.7.1",
+		"@emotion/styled": "^11.6.0",
+		"@mui-treasury/components": "^1.10.1",
+		"@mui/icons-material": "^5.3.1",
+		"@mui/lab": "^5.0.0-alpha.68",
+		"@mui/material": "^5.3.1",
+		"@testing-library/jest-dom": "^5.16.1",
```

Comment:
I don't think testing-libraries are dependencies, same for @types, that should be in dev-devencies.